src/styles/blocks/month.scss

```diff
@@ -0,0 +1,40 @@
+.month {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $square-gap;
+  width: $container-width;
+
+  @each $day, $indentation in $week {
+    &--start-day--#{$day} :first-child {
+      margin-left: $indentation;
+    }
+  }
+
+  @for $last-day from $min-month-length to $max-month-length {
+    &--month-lenght--#{$last-day} :nth-child(#{$last-day + 1}) {
+      display: none;
+    }
+  }
```

Comment:
this scss rule doesn't work properly
https://www.loom.com/share/cd4432c86c1b4ee885f631cb6849c45d

you should apply display none to month__day :nth-child inside the parent block (use nesting)